webapp/app/reducer.js

```diff
@@ -34,26 +26,6 @@ const actionHandlers = {
 
   // ====== saving
   [appSavingUpdate]: (state, { saving }) => AppState.assocSaving(saving)(state),
-
-  // ====== sideBar
-  [appSideBarOpenedUpdate]: (state, { sideBarOpened }) => AppState.assocSideBarOpened(sideBarOpened)(state),
-
-  // ====== app job
-  [appJobStart]: (state, { job, onComplete, autoHide }) => AppState.startJob(job, onComplete, autoHide)(state),
-
-  [appJobActiveUpdate]: (state, { job }) => AppState.updateActiveJob(job)(state),
-
-  // ===== app errors
-  [appErrorCreate]: (state, { error }) => AppState.assocAppError(error)(state),
-
-  [appErrorDelete]: (state, { error }) => AppState.dissocAppError(error)(state),
-
-  [systemErrorThrow]: (state, { error }) => AppState.assocSystemError(error)(state),
```

Comment:
[systemErrorThrow] should be left in this reducer and removed from appErrors/reducer